api/docs/api.yml

```diff
@@ -4,23 +4,29 @@ info:
   title: Octopus
```

Comment:
Did you manually do all this?

Reply:
I did - it seemed like the tools that help you out with it would always take some amount of manual correction / coercion along the way too, so I just went with having full control by writing it manually. I'm using Swagger hub which is handy because it will point out any errors/warnings as you go.